apis/compute/v1beta1/computeforwardingrule_types.go

```diff
@@ -88,25 +87,25 @@ type ForwardingruleServiceDirectoryRegistrations struct {
 
 type ForwardingruleTarget struct {
 	// +optional
-	ServiceAttachmentRef *refs.ComputeServiceAttachmentRef `json:"serviceAttachmentRef,omitempty"`
+	ServiceAttachmentRef *v1alpha1.ResourceRef `json:"serviceAttachmentRef,omitempty"`
```

Comment:
We do lose the ability to have nice per-ref-type comments now, IIUC

Reply:
I do not think that's required for existing Beta resources(?), ideality we should not introduce any new changes to Beta CRDs?